src/instructlab/cli/model/evaluate.py

```diff
@@ -135,6 +133,29 @@
     is_flag=True,
     help="Print serving engine logs.",
 )
+@click.option(
```

Comment:
shouldn't these be more specific like `dk-input-questions` and `dk-output-file-formats`? or not. if so, you can make this work by doing:

```
@click.option(
"--dk-input-questions",
"input_questions",
type=click.STRING,
cls=clickext.ConfigOption,
config_sections="dk_bench",
)

```

explicitly specifying the variable name will ensure it reads from the right spot in the config

Reply:
`--dk-input-questions`, we probably don't want to do since we may have other evals in the future also reading from files. `--dk-output-file-formats` doesn't really work either, since all of the evals should be able to generate a report and allow you to pick which format you want it in.